test/manifold/executor_test.clj

```diff
@@ -58,8 +58,8 @@
   (let [tf (thread-factory)]
     (.newThread tf (constantly nil)))
```

Comment:
Are these lines supposed to test something? Should they check that a Thread is returned or something?

Reply:
`is` added.
The aim was to proved it doesn't return an Exception.